test/Tests/src/Data/Text_Spec.enso

```diff
@@ -134,6 +134,10 @@ spec =
             Ordering.compare common_prefix+complex_letter_3+later_suffix  common_prefix+complex_letter_1+earlier_suffix . should_equal Ordering.Greater
             Ordering.compare common_prefix+complex_letter_1+later_suffix  common_prefix+complex_letter_2+earlier_suffix . should_equal Ordering.Greater
 
+        Test.specify "normalizes correctly" <|
+            'aśb'.normalize . should_equal 'as\u0301b'
+            'as\u0301b'.normalize . should_equal 'as\u0301b'
```

Comment:
This test checks nothing, because in Enso `==` is performed modulo Unicode normalization.

That is, `'aśb' == 'as\u0301b'` will _already_ yield true.

To check this works, we should check the internal representation of the strings, by comparing their vectors of codepoints.

```suggestion
        Test.specify "normalizes correctly" <|
            'aśb'.normalize.codepoints . should_equal 'as\u0301b'.codepoints
            'as\u0301b'.normalize.codepoints . should_equal 'as\u0301b'.codepoints
            # Sanity check
            'aśb'.codepoints . should_not_equal 'as\u0301b'.codepoints
```

Reply:
Ah, right, I wrote these tests before I understood it was normalizing during comparison. Changed to check codepoints.